### DISCUSSION TITLE
Sizing Documentation

### DISCUSSION THREAD

**Discussion by cdancy** [vectordotdev/helm-charts#330] [General]
Hey All,

Looking for general sizing documentation surrounding vector (i.e. num pods, memory, cpu, etc..) that you all may/may-not have :) General documentation suggests vector can handle loads but is that with the OOTB config? 

Now I completely understand that sizing, and things of this nature, are context specific but maybe you all have _some documentation_ on how to get some basic sizing configs in place given number of logs over some period of time.

Thanks,
Chris

**Comment by jszwedko** [comment]
Hi @cdancy !

You can see some documentation around this here: https://vector.dev/docs/setup/going-to-prod/

**Reply by cdancy** [reply]
I knew I saw this somewhere :) 

Ok ... let me see if this gets us what we need and I'll get back. Thanks @jszwedko !
